js/src/setup-mc/setup-stepper/setup-paid-ads/setup-paid-ads.js

```diff
@@ -150,7 +161,13 @@ export default function SetupPaidAds() {
 				text={ text }
 				loading={ completing === ACTION_SKIP }
 				disabled={ disabledSkip }
-				onClick={ finishOnboardingSetup }
+				onClick={
+					// Show the skip modal only for the "Skip this step for now" button only.
+					showPaidAdsSetup
+						? finishOnboardingSetup
+						: handleShowSkipPaidAdsConfirmationModal
+				}
+				// TODO: might want to review eventName and eventProps
```

Comment:
@joemcgill Shall we keep those props as is?

Reply:
Good question. I think we'll want to move that eventName to the new "Complete setup" button in the AYS modal, but the event name for this will likely need an update. Leave the TODO: comment for now and we can address that before the feature is merged.

Reply:
👍